config/configmaps/install-log-regexes-configmap.yaml

```
@@ -82,6 +82,16 @@ data:
       - "platform\\.aws\\.userTags.*: Invalid value:.*value contains invalid characters"
       installFailingReason: InvalidAWSTags
       installFailingMessage: You have specified an invalid AWS tag value. Verify that your tags meet AWS requirements and try again.
+    - name: AWSSubnetDoesNotExist
+      searchRegexStrings:
+      - "The subnet ID '[a-z0-9-]*' does not exist"
```

This is probably okay, but I couldn't find a definitive format spec for a subnet ID beyond "String". (The example [here](https://docs.aws.amazon.com/AWSEC2/latest/APIReference/API_CreateSubnet.html) has uppercase letters in it FWIW.) We can be pretty sure it won't have single quotes, though, so a safer regex might be:
```suggestion
      - "The subnet ID '[^']*' does not exist"
```
(single quote, stuff-that-is-not-single-quote, single-quote)